index.js

```diff
@@ -2,6 +2,24 @@
 
 module.exports = Result;
 
+Result.just = function just(value) {
+    return new Result(null, value);
+};
+
+Result.error = function error(err) {
+    return new Result(err, null);
+};
+
+Result.lift = function lift(func) {
+    return function rezultLifted() {
+        try {
+            return Result.just(func.apply(this, arguments));
+        } catch(err) {
+            return Result.error(err);
+        }
+    };
+};
```

Comment:
Let’s put these after the Result constructor declaration. I :heart: lift. I’ve been moving away from just and error since new Result(err) and new Result(null, value) are fine.